src/components/data/data-utils.js

```diff
@@ -376,9 +376,8 @@ export const EntityUploader = ({ onSuccess, onDismiss, namespace, name, entityTy
   const [recordType, setRecordType] = useState(undefined);
   const [recordTypeInputTouched, setRecordTypeInputTouched] = useState(false);
 
-  // TODO: https://broadworkbench.atlassian.net/browse/WOR-614
-  // This value is mostly hard-coded for now for Azure public preview. Once WOR-614 is complete, this value can be dynamically updated
-  const regionLabelToDisplay = isGoogleWorkspace ? 'US' : getRegionLabel(defaultAzureRegion);
+  const regionLabelToDisplay = isGoogleWorkspace ? 'US' : getRegionLabel(region);
```

Comment:
Why is it always US for Google workspaces?

Reply:
Honestly, it was so weird that they were both hardcoded in completely different ways that it seemed like it had to be deliberate to keep Google that way but leave Azure open for future change - but I have no real evidence for that assumption.

Reply:
Update: "keep hardcoding in GCP bc it uploads to the Rawls service which is only on uscentral-1".

Reply:
Thanks for the update! Could you add a code comment to this effect?

Reply:
Ah yeah I was going to and then got distracted and forgot!  Thanks!  It's there now.